src/Common/src/Interop/Windows/sspicli/SafeDeleteContext.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Authentication.ExtendedProtection;
+
+namespace System.Net.Security
+{
+    //
```

Comment:
Oh, I see, you just separated it out of SecuritySafeHandles.cs. Ok.

---
In reply to: [87213631](https://github.com/dotnet/corefx/pull/13502#discussion_r87213631) [](ancestors = 87213631)